swap_meet/vendor.py

```diff
@@ -1,2 +1,55 @@
+from .item import Item
```

Comment:
We don't need to import Item into this file since we never use it by name. If you have a look in VSCode, Item should be slightly greyed out and if you hover over it you'll see the message "Item is not accessed"